.github/workflows/appsignals-e2e-eks-test.yml

```diff
@@ -116,49 +116,51 @@ jobs:
 
       # Application pods need to be restarted for the
       # app signals instrumentation to take effect
-      - name: Restart the app pods
-        run: kubectl delete pods --all -n ${{ env.SAMPLE_APP_NAMESPACE }}
+      - name: Restart the app pods and wait for the endpoints to come online
+        id: restart-pod
+        uses: nick-fields/retry@v2
+        with:
+          max_attempts: 3
+          retry_on: error
+          timeout_minutes: 6
```

Comment:
What is this timeout for? Is it the time limit for the all the commands to run successfully before retrying?

Reply:
Yup it's the time limit for all the commands to run. On a successful run it takes less than 2 minutes, but on failed runs it seems like it takes ~6 minutes.